packages/teleport/src/Kubes/EventDialog/EventDialog.tsx

```diff
@@ -0,0 +1,100 @@
+/*
+Copyright 2021 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React from 'react';
+import Dialog, {
+  DialogFooter,
+  DialogHeader,
+  DialogTitle,
+  DialogContent,
+} from 'design/Dialog';
+import { Text, Box, ButtonSecondary } from 'design';
+import cfg from 'teleport/config';
+import TextSelectCopy from 'teleport/components/TextSelectCopy';
+
+function EventDialog(props: Props) {
+  const { onClose, user, kubeName } = props;
+  const { hostname, port } = window.document.location;
+  const host = `${hostname}:${port || '443'}`;
+
+  return (
+    <Dialog
+      dialogCss={dialogCss}
+      disableEscapeKeyDown={false}
+      onClose={onClose}
+      open={true}
+    >
+      <DialogHeader>
+        <DialogTitle typography="h3" caps={true}>
+          connect to kubernetes cluster
+        </DialogTitle>
+      </DialogHeader>
+      <DialogContent minHeight="240px" flex="0 0 auto">
+        <Box>
+          <Box mb={4}>
+            <Text bold as="span">
+              Step 1
+            </Text>
+            {' - Login to Teleport'}
+            <TextSelectCopy
+              mt="2"
+              text={`tsh login --proxy=${host} --auth=${cfg.getAuthType()} --user=${user}`}
+            />
+          </Box>
+          <Box mb={4}>
+            <Text bold as="span">
+              Step 2
```

Comment:
```suggestion
              *Step 2
```
an asterik to indicate to user there is a note for user to read before applying step 2